openapi/workflow_execution_service.swagger.yaml

```diff
@@ -495,10 +540,10 @@ definitions:
       workflow_url:
         type: string
         description: |-
-          OPTIONAL
-          The workflow CWL or WDL document, must provide either this or workflow_descriptor. When a base64 encoded gzip of
-          workflow descriptor files is offered, the `workflow_url` should be set to the relative path
-          of the main workflow descriptor.
+          REQUIRED
+          The workflow CWL or WDL document.
+          When workflow attachments files are provided, the `workflow_url` may be a relative path
+          corresponding to one of the attachments.
```

Comment:
Would it be clearer to just say that `workflow_url` should match one of the filename parameters in the Content-Disposition headers?